### PR TITLE
corrected the grammatical mistakes

### LINKED ISSUE #49: Fix grammatical mistakes
### Is your feature request related to a problem? Please describe.

Fix grammatical mistakes under [this](https://github.com/anitab-org/documentation/blob/master/ways-to-contribute.md#ways-to-contribute) section.

### PR DESCRIPTION
**Pull request by unnati914**
I have corrected the grammatical errors in different files, solves issue#49

### Description
 I have corrected the grammatical errors in different files. Eg some spelling mistakes 

Fixes #49 

### Type of Change:

- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have  provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by devkapilbansal** [comment]
Hi @unnati914 please edit the description above and made necessary changes. Also, we do have a commit message guideline that should be followed.

P.S. :- I can't see any document file change. Only changes shown are in binary files. Can you please re check it?

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@unnati914 the file asked in issue is still not updated. The section is :- https://github.com/anitab-org/documentation/blob/master/ways-to-contribute.md#ways-to-contribute

File: DESIGN_CONTRIBUTION_GUIDELINES.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ Open Source Ambassadors is at least half year contribution program organised by
...
-- Attend product roadmap in assigned project
-- Review existing designs
-- Design new feature, redesign existing features (if ticket exist and assigned)
-- Create and maintain metrics document and guard consistency in the project
-- Review UI coding PR’s (test if everything looks as expected)
-- Attend design session to discuss challenges in design within our projects
-- Create User Journey wireframes
-- Meet on a monthly basis, with all designers to discuss processes
+- Attend product roadmap in assigned project.
+- Review existing designs.
```
Comment (epicadk) [inline DESIGN_CONTRIBUTION_GUIDELINES.md:20]: This change isn't required.

File: README.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ If you are here, it probably means you are either looking for a place to start c
 
 ## Contributing
 
-We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following list.
+We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following lists.
```
Comment (epicadk) [inline README.md:11]:
```suggestion
We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend the following :-
```

File: README.md [hunk 3]
```diff
@@ -8,14 +8,14 @@ If you are here, it probably means you are either looking for a place to start c
...
 
 ## Documentation
 
-Below is a list of documents currently in this repository. If none of these documentation helps, feel free to reach out to us either by asking your question on Zulip or creating an issue requesting the information you need.
+Below is a list of documents currently in this repository. If none of these documentations helps, feel free to reach out to us either by asking your question on Zulip or creating an issue requesting the information you need.
```
Comment (epicadk) [inline README.md:18]:
```suggestion
Below is a list of documents currently in this repository. If none of these documents help, feel free to reach out to us either by asking your question on Zulip or creating an issue requesting the information you need.
```

File: our-blog.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ Make sure to provide us with following details:
 3. Select "Add to publication" option;
 4. Select the AnitaB.org Open Source (you will see this in case you are a writer);
     <img src="https://user-images.githubusercontent.com/11148726/87883600-78940a00-ca00-11ea-9534-49eabeff2f16.png" alt="AnitaB.org open source option in the dropdown" width="600"/>
-5. Click submit!
+5. Click submit
```
Comment (epicadk) [inline our-blog.md:46]: @isabelcosta was the `!` for dramatic effect :P ?

File: quality-assurance.md [hunk 5]
```diff
@@ -88,11 +88,11 @@
 1. Choose a test scenario in the project you’ve chosen and execute it on either the PR or deployed version that is being tested.
    * Open issues under the respective github repo if you find any bugs.
 
-2. Search on TestQuality if the Test Case is already documented. If not, document the testcase on TestQuality under the respective Test Plan and Test Suite. Here is a [demo](https://drive.google.com/file/d/1pkxCAPR9G3lYjUf_Ym8bmuglDWBdEU2c/view?usp=sharing).
+2. Search on Test Quality if the Test Case is already documented. If not, document the testcase on TestQuality under the respective Test Plan and Test Suite. Here is a [demo](https://drive.google.com/file/d/1pkxCAPR9G3lYjUf_Ym8bmuglDWBdEU2c/view?usp=sharing).
```
Comment (epicadk) [inline quality-assurance.md:91]: Change not required

**Review by epicadk — CHANGES_REQUESTED** [review]

File: ways-to-contribute.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 This document lists the many ways **you** as a member of this community can contribute to our community! You can be a newcomer to the community or someone who is with us for a while :)
```
Comment (devkapilbansal) [inline ways-to-contribute.md:3]:
```suggestion
This document list many ways **you** as a member of this community can contribute to our community! You can be a newcomer to the community or someone who is with us for a while :)
```

File: ways-to-contribute.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
...
 
-We encourage you to contribute in any way you can and would like! You can contribute in many categories such as quality assurance, design, coding, outreach/research, documentation, ...
+We encourage you to contribute in any way you can and would like! You can contribute in many categories such as quality assurance, design, coding, outreach/research, documentation etc.
```
Comment (devkapilbansal) [inline ways-to-contribute.md:5]:
```suggestion
We encourage you to contribute in any way you can and would like it! You can contribute in many categories such as quality assurance, design, coding, outreach/research, documentation etc.
```

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@unnati914 there are some more errors that can be fixed. I will suggest to use some grammar checker tool for that. :smiley:

File: ways-to-contribute.md (hunk 6 above)
Comment (epicadk) [reply ways-to-contribute.md:3]: The seems fine here : /

File: ways-to-contribute.md (hunk 7 above)
Comment (epicadk) [reply ways-to-contribute.md:5]: I don't think `it` is required here

File: our-blog.md (hunk 4 above)
Comment (isabelcosta) [reply our-blog.md]: loool @epicadk maybe maybe. Yes for encouragement. like a "hit publish and that's it! 🥳 "
it is a style choice :P

File: README.md [hunk 8]
```diff
@@ -8,14 +8,14 @@ If you are here, it probably means you are either looking for a place to start c
 
 ## Contributing
 
-We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following list.
+We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following:-
```
Comment (isabelcosta) [inline README.md:11]:
```suggestion
We are glad to know that you would like to contribute to our open source projects. If you are new to our community, we strongly recommend getting through the following:
```

**Comment by isabelcosta** [comment]
@unnati914 did you close because you don't want to continue working on this task?

**Comment by unnati914** [comment]
I had open a new PR. Because I had some conflicts opening a new PR. I had worked on this issue

**Pull request closed without merging**